Capstone Project/ViewControllers/Workouts/CurrentWorkoutViewController.m

```diff
@@ -89,10 +80,17 @@ -(void)getExercises{
             [alert addAction:defaultAction];
             [self presentViewController:alert animated:YES completion:nil];
         }
-        [self.refreshControl endRefreshing];
     }];
 }
 
+- (IBAction)didTapAdd:(id)sender {
+    NSArray *repsPerSet = [self.repsTextField.text componentsSeparatedByString:@"/"];
+    [Exercise newExercise:self.searchBar.text withWeight:[self.weightTextField.text floatValue] withSet1:[[repsPerSet objectAtIndex:0] integerValue] withSet2:[[repsPerSet objectAtIndex:1] integerValue] withSet3:[[repsPerSet objectAtIndex:2] integerValue] withSet4:[[repsPerSet objectAtIndex:3] integerValue] withSet5:[[repsPerSet objectAtIndex:4] integerValue] withWorkout:self.selectedWorkout withCompletion:^(BOOL succeeded, NSError * _Nullable error){
```

Comment:
Directly accessing index without checking the size of the array is a recipe for crash